src/Button/Button.Component.js

```diff
@@ -238,7 +238,7 @@ const clickBtnHandler = btn => {
 
             <h2>Button States</h2>
             <Description>
-                There are three states: <code>normal</code> (default), <code>selected</code>, and <code>disabled</code>.
+                There are three states: `normal` (default), `selected`, and `disabled`.
```

Comment:
@mobot11 is this still accurate? Or is it just default, `selected` and `disabled`?

Reply:
Looks like a button can be both selected and disabled
```javascript
    const buttonClasses = classnames(
        {
            'fd-button': !option,
            [`fd-button--${option}`]: !!option,
            [`fd-button--${type}`]: !!type,
            'fd-dropdown__control': dropdown,
            'fd-button--compact': compact,
            [`sap-icon--${glyph}`]: !!glyph,
            'fd-global-nav__btn': navbar,
            'is-selected': selected,
            'is-disabled': disabled
        },
        className
    );
```

But adding normal as a prop would have no effect on how the component actually renders. However fundamentals does describe the default state as normal in their documentation so I'd leave this as is to stay synced up with fundamentals https://sap.github.io/fundamental/components/button.html#standard-button-state